lein-light-nrepl/src/lighttable/nrepl/eval.clj

```diff
@@ -70,7 +80,9 @@
      (and (string? res) (:verbatim opts)) res
      :else (pr-str res))))
 
-(defn truncate [v]
+(defn truncate
```

Comment:
I can't find any references to this function. I think it's safe to remove, ignore for now, or just change the docstring to reflect that it's not used.